Sources/xcresultparser/JunitXML.swift

```diff
@@ -100,16 +102,11 @@ public struct JunitXML {
         
         if testReportFormat != .sonar {
             if let date = testAction?.startedTime {
-                let dateFormatter = ISO8601DateFormatter()
-                testsuites.addAttribute(name: "timestamp", stringValue: dateFormatter.string(from: date))
                 if let ended = testAction?.endedTime {
                     let duration = ended.timeIntervalSince(date)
                     testsuites.addAttribute(name: "time", stringValue: numFormatter.unwrappedString(for: duration))
                 }
             }
-            if let runDestination = testAction?.runDestination {
```

Comment:
Why did you remove this code "for good"? The information about the run destination (runDestinationXML()) is not used at all anymore.
Isn't it useful to see, on which environment the tests were run?

Reply:
Oh, I see it now in your PR description: "Fix junit output format to conform to the XSD"
I will leave the 'runDestinationXML' method in for now and just add a comment, that it is not in use.